src/dvclive/live.py

```diff
@@ -449,6 +446,19 @@ def log_artifact(
                     )
 
     def cache(self, path):
+        if self._inside_dvc_exp:
+            from dvc.exceptions import OutputNotFoundError
+
+            msg = f"Skipping dvc add {path} because `dvc exp run` is running."
+            try:
+                self._dvc_repo.find_outs_by_path(path)
+                msg += " It is already being tracked automatically."
+                logger.info(msg)
+            except OutputNotFoundError:
```

Comment:
Let's add a test for this and for the logging

Reply:
To test the logging statements? Do you think it's needed, and do you have an idea for how to do it?

Reply:
> To test the logging statements? 

To test the try/except logic via the logging statements.

> Do you think it's needed, and do you have an idea of how to do it?

https://github.com/iterative/dvclive/blob/b2a66c29286a8afc3eedd8082dbe5ed4f0c39bbd/tests/test_main.py#L326

Reply:
It is not strictly needed, but nice to have it tested using the non-mocked `dvc_repo` fixture

Reply:
Before I bother 😄 , let's agree on the behavior

Reply:
I think the behavior is ok

Reply:
Tweaked the messages and added tests.

If it is a stage output, the message is:
```
INFO: Skipping dvc add {path} because `dvc exp run` is running.
It is already being tracked automatically.
```

If it is another kind of output:
```
WARNING: Skipping dvc add data because `dvc exp run` is running.
To track it automatically during `dvc exp run`:
1. Run `dvc exp remove data.dvc` to stop tracking it outside the pipeline.
3. Add it as an output of the pipeline stage.
```

If it is not an output:
```
WARNING: Skipping dvc add data because `dvc exp run` is running.
To track it automatically during `dvc exp run`, add it as an output of the pipeline stage.
```

Reply:
@dberenbaum I am missing those changes in the P.R.

Reply:
Well, I nuked my local repo recently for some reason I can't remember, so looks like I need to redo these changes 🤦